features/step_definitions/authorization_requests_steps.rb

```diff
@@ -87,6 +87,13 @@
   )
 end
 
+Quand('je renseigne les infos de minimales du projet') do
```

Comment:
```suggestion
Quand('je renseigne les infos minimales du projet') do
```

limite en vrai j'utiliserai celui des infos de base

Reply:
Zut, je ne voulais pas commit ce changement qui ne sert à rien... surtout que les infos de base ne remplissent que intitulé et description dans features